packages/odyssey-react-mui/src/SearchField.tsx

```diff
@@ -166,6 +170,7 @@ const SearchField = forwardRef<HTMLInputElement, SearchFieldProps>(
               <SearchIcon />
             </InputAdornment>
           }
+          tabIndex={tabIndex}
```

Comment:
Same comment here as the one above with conditionally providing this prop

Reply:
This actually doesn't work. Need to pass as `inputProps`. So, won't be applied if undefined.

Reply:
Understood - thanks!